spark/dl/src/test/scala/com/intel/analytics/bigdl/utils/intermediate/BlasToDnnSpec.scala

```diff
@@ -82,4 +83,35 @@ class BlasToDnnSpec extends BigDLSpecHelper {
     Equivalent.nearequals(outDnn, outBlas, 1e-6) should be(true)
     Equivalent.nearequals(gradInputDnn, gradInputBlas, 1e-6) should be(true)
   }
+
+  "resnet50 blas to dnn" should "work properly" in {
+    System.setProperty("bigdl.engineType", "mkldnn")
+    val batchSize = 2
+    val classNum = 1000
+    RandomGenerator.RNG.setSeed(1000)
+    val input = Tensor[Float](Array(batchSize, 3, 224, 224)).apply1(_ =>
+      RandomGenerator.RNG.uniform(0.1, 1.0).toFloat)
+    var gradOutput = Tensor[Float](batchSize, classNum).apply1(_ =>
+      RandomGenerator.RNG.uniform(1.0, 1000.0).toFloat)
+
+    val blas = ResNet.graph(classNum,
+      T("shortcutType" -> ShortcutType.B, "depth" -> 50,
+        "optnet" -> false, "dataset" -> DatasetType.ImageNet)).asInstanceOf[StaticGraph[Float]]
+    val irBlas = blas.toIRgraph()
+
+    irBlas.build()
+    val outBlas = blas.forward(input).toTensor[Float]
+    val outDnn = irBlas.forward(input).toTensor[Float]
+
+
+    gradOutput.resizeAs(outBlas).apply1(_ =>
+      RandomGenerator.RNG.uniform(1.0, 1000.0).toFloat)
+
+    val gradInputBlas = blas.backward(input, gradOutput).toTensor[Float]
+
+    val gradInputDnn = irBlas.backward(input, gradOutput).toTensor[Float]
+    val gradInputTensor = Tensor[Float]().resize(gradInputDnn.size()).copy(gradInputDnn)
+
+    Equivalent.nearequals(outDnn, outBlas, 1e-6) should be(true)
```

Comment:
We should clear the property at the end. Sometimes it will influence other test cases.

Reply:
yeah, good idea